### DISCUSSION TITLE
Fullcone NAT里的高性能和兼容，具体是什么差异？

### DISCUSSION THREAD

**Discussion by Travel2Here** [coolsnowwolf/lede#11041] [Q&A / 快速问答]
rt，能否展开说说？

**Comment by hpx502766238** [comment]
@coolsnowwolf

**Comment by Firsgith** [comment]
bilibili韩风talk在openwrt防火墙那一期又讲。具体我忘了，所以这里就不说了

**Reply by hpx502766238** [reply]
没找到.....现在有个问题就是，我由于需要用510内核所以用的lean lede20220716的版本，但这个版本无法开启全锥形高性能模式，只能开兼容，所以我想弄清楚它们到底有什么区别
